internal/test_helper.go

```diff
@@ -0,0 +1,37 @@
+package internal
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/autograde/quickfeed/database"
+	"github.com/autograde/quickfeed/log"
+)
+
+// TestDB returns a test database and close function.
+// This function should only be as a test helper.
+func TestDB(t *testing.T) (database.Database, func()) {
+	t.Helper()
+
+	f, err := ioutil.TempFile(t.TempDir(), "test.db")
```

Comment:
This line returns `t.TempDir undefined (type *testing.T has no field or method TempDir)` for me.

Reply:
Ah, right. We need to upgrade to a newer Go version.